### DISCUSSION TITLE
Delete more than one songs from queue (using select)

### DISCUSSION THREAD

**Discussion by Retrofreak** [rern/rAudio#806] [Fixed Issues] [answered]
When I go to Playlist, click the "Minus"-button, chose "Select": Then it is possible to remove a maximum of two entries from the queue. Then the browser (chrome, firefox, opera) automatically refreshes immediately? Keeping the select options alive for more than a few seconds would be great. Or am I doing something wrong?

**Comment by EarsofTin** [comment]
I have the same screen behavior.  My workaround is to press the musical note icon of a particular track found at the far left followed by "remove".  Maybe faster for just a few deletions.  But this method is prone to FFE (fat finger error). If I hit the text of the song title instead of the note icon the song I'm trying to avoid immediately starts playing.

**Comment by rern** [comment, marked as answer]
Fixed with today's update.

**Reply by Retrofreak** [reply]
Indeed! Great, thank you!
